filter/kubernetes_filter.go

```diff
@@ -107,7 +107,8 @@ func (filter *KubernetesContainerFilter) getWorkspacePods() (*v1.PodList, error)
 		return nil, errors.New("unable to get current workspace id")
 	}
 
-	filterOptions := metav1.ListOptions{LabelSelector: WsIdLabel + "=" + workspaceID, FieldSelector: "status.phase=Running"}
+	labelSelector := fmt.Sprintf("%s=%s", cfg.PodLabelSelector, workspaceID)
```

Comment:
It's still che strong dependent solution.
WDYT about the following abstract way to configure pod selector https://github.com/JPinkney/che-machine-exec/compare/workspace-id-configurable...sleshchenko:workspace-id-configurable-proposal?expand=1